src/chapters/data-manipulation/exercises.rst

```diff
@@ -1,2 +1,24 @@
 Exercises: Data Manipulation
-============================
\ No newline at end of file
+============================
+
+.. _dataManipulationNotebook:
+
+Getting Started
+---------------
+
+| The exercises have been combined with the reading for this chapter.  
```

Comment:
```suggestion
The exercises have been combined with the reading for this chapter.   This is a reminder to submit both of your notebooks.  
```

Reply:
could link back to the reading in this line?

Reply:
maybe internal link on the word "reading"?  or even the entire phrase "reading for this chapter"

Reply:
In trying to keep the formant consistent throughout the chapters - There is a link back to the Readings Section( line 15). We aren't forking and cloning notebooks.  We are forking and cloning GitHub repositories, which contain the notebooks.

Reply:
Reworked both data-manipulation.rst and exercises.rst, thoughts?